DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Banner.Edit.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@model OrchardCore.ContentFields.ViewModels.EditContentPickerFieldViewModel
+@using OrchardCore.ContentFields.Settings
+@using Newtonsoft.Json;
+@using Newtonsoft.Json.Serialization;
+@using OrchardCore.ContentManagement
+@using OrchardCore.ContentManagement.Records
+@using YesSql;
+@inject ISession Session
+
+@{
+    var settings = Model.PartFieldDefinition.GetSettings<ContentPickerFieldSettings>();
+    var partName = Model.PartFieldDefinition.PartDefinition.Name;
+    var fieldName = Model.PartFieldDefinition.Name;
+    var searchUrl = Url.RouteUrl(new { area = "DFC.ServiceTaxonomy.ContentPickerPreview", controller = "BannerContentPickerAdmin", action = "SearchContentItems", part = partName, field = fieldName });
+    var vueElementId = $"ContentPicker_{partName}_{fieldName}_{Guid.NewGuid().ToString("n")}";
+    var multiple = settings.Multiple.ToString().ToLowerInvariant();
+
+    var selectedBannerItems = new List<VueMultiselectBannerItemViewModel>();
+
+    var bannerContentItems = await Session.Query<ContentItem, ContentItemIndex>().Where(x => x.ContentType == "Banner" && x.Latest).ListAsync();
+
+    foreach (var selectedItem in Model.SelectedItems)
+    {
+        selectedBannerItems.Add(new VueMultiselectBannerItemViewModel
+        {
+            Id = selectedItem.Id,
+            DisplayText = selectedItem.DisplayText,
+            HasPublished = selectedItem.HasPublished,
+            IsActive = bannerContentItems.FirstOrDefault(x => x.ContentItemId == selectedItem.Id).Content.Banner.IsActive.Value,
```

Comment:
FirstOrDefault could return null?  or any property after.. You may want to add null propagation and default it to false.

Reply:
Not possible to get null at this stage. But Updates code to handle null